install-manager/src/main/java/cm/aptoide/pt/install_manager/repository/TaskInfoRepository.kt

```diff
@@ -30,5 +31,7 @@ interface TaskInfoRepository {
    * @param packageName - a package name
    */
   suspend fun removeAll(packageName: String)
+
+  companion object
```

Comment:
Why did you need to set up these?
I know they can be used as TaskInfoRepository.Companion, but why doesnt it have anthing ? 
I also noticed it is only being used in the mocks. Does using it this way allow you to define whatever fields you want for the companion object?